FirebaseInstallations/Source/Tests/Unit/FIRInstallationsIDControllerTests.m

```diff
@@ -1504,4 +1446,76 @@ - (void)expectBackoffEvent:(FIRInstallationsBackoffEvent)event
   OCMVerifyAll(self.mockBackoffController);
 }
 
+- (void)assertGetInstallation_NewFIDCreatedAndRegistered {
```

Comment:
nit:should method name be camel case without "_"?

Reply:
I added "_" in some tests to the very long names to break the name to logical part to improve readability. I struggled to come up with anything better.

Reply:
That's probably fine. Just want to call out the style guide. 
I'm okay if it leaves like this.